keyboards/projectkb/signature87/signature87.c

```diff
@@ -0,0 +1,21 @@
+#include "signature87.h"
+
+void matrix_init_board(void){
+    setPinOutput(B11);
+    setPinOutput(B5);
+}
+
+
+void led_set_kb(uint8_t usb_led) {
+    if (IS_LED_ON(usb_led, USB_LED_CAPS_LOCK)) {
+        writePinLow(B11);
+    } else {
+        writePinHigh(B11);
+    }
+    if (IS_LED_ON(usb_led, USB_LED_SCROLL_LOCK)) {
+        writePinLow(B5);
+    } else {
+        writePinHigh(B5);
+    }
+    led_set_user(usb_led);
+}
```

Comment:
Please change this to:
```c
bool led_update_kb(led_t led_state) {
    if (led_update_user(led_state)) {
        writePin(B11, !led_state.caps_lock);
        writePin(B5, !led_state.scroll_lock);
    }
}
```
This will allow user keymaps to override this for eg. layer indication.

Reply:
@fauxpark Isn't there supposed to be a `return true` at the end?

Reply:
...yes. I need more sleep. And to make this code a saved reply, I think 😁